medicines/doc-index-updater/src/pars_upload.rs

```diff
@@ -221,7 +221,7 @@ fn product_form_data_to_blob_metadata(
     let product_names = vec![product_name];
 
     let title = get_field_as_string(&fields, "title")?;
-    let pl_number = get_field_as_string(&fields, "license_number")?;
```

Comment:
The interface between the frontend and the API has slightly changed by renaming this field but I've run the E2E tests locally (without the mock server) to make sure it still works fine